jobs/mongodb_migration/src/mongodb_migration/collector.py

```diff
@@ -291,4 +292,7 @@ def get_migrations(self) -> list[Migration]:
             MigrationRemoveFieldFromJob(
                 field_name="finished_at", version="20231201112600", description="remove 'finished_at' field from queue"
             ),
+            MigrationAddRetriesToCacheResponse(
+                version="20240104085000", description="add 'retries' field to cache records"
+            ),
```

Comment:
when deploying, we might want to stop the workers, in order to be sure to add the field for all the cache entries

Reply:
Makes sense, I will take care of it.